interfaces/media-source.idl

```diff
@@ -55,25 +56,29 @@ interface SourceBuffer : EventTarget {
                     attribute EventHandler        onabort;
     undefined appendBuffer (BufferSource data);
     undefined abort ();
+    undefined changeType (DOMString type);
     undefined remove (double start, unrestricted double end);
 };
 
-[Exposed=Window]
+[Exposed=(Window,DedicatedWorker)]
 interface SourceBufferList : EventTarget {
     readonly        attribute unsigned long length;
                     attribute EventHandler  onaddsourcebuffer;
                     attribute EventHandler  onremovesourcebuffer;
     getter SourceBuffer (unsigned long index);
 };
 
+[Exposed=(Window,DedicatedWorker)]
```

Comment:
This will cause the "Partial interface AudioTrack: valid exposure set" test to fail, see https://github.com/w3c/media-source/issues/280.

But since only tests in MSE fail, I don't think that should block this PR.